### DISCUSSION TITLE
How to set a footer tag as body text in pdf?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#2666] [General]
How to set a footer tag as body text in pdf?

**Comment by JorjMcKie** [comment]
This is not supported yet.

**Comment by nissansz** [comment]
When exporting pdf to docx, sometimes, may generate docx with footer/headers etc. How can I export docx without headers/footers, export all content as body text?

**Reply by JorjMcKie** [reply]
This depends on the tool you are using.
PDFs may or may not contain tags that identify headers and footers, and your tool may or may not understand that ...

**Comment by nissansz** [comment]
How to use pymupdf to export to normal docx without header/footer style?

**Reply by JorjMcKie** [reply]
We do not currently support the export to DOCX.
There exists a package on PyPI which can do that, and it is based on PyMuPDF, https://pypi.org/project/pdf2docx/.

**Reply by nissansz** [reply]
I tried it. It exports as tables, not good.

**Reply by JorjMcKie** [reply]
well ... 🤷‍♂️
